stream/personalization.py

```diff
@@ -0,0 +1,122 @@
+class Personalization(object):
```

Comment:
Can we have methods with very explicit names to do what we support now and add some room to perform arbitrary REST calls?

I find the `.get()` method too generic/vague. I guess that's because we use it for multiple things (feeds, follow_recommendation, ..?)

```python
# have this methods implemented with actual code and with docs 
personalization.feed.get(user_id="tommaso")
personalization.follow_recommendation.get(user_id="tommaso")

# support this via magic methods, accepts kwargs and return json
personalization.popular_cats_in_my_hood.get(**kwargs)
```

Reply:
I think we should only expose the generic get API since these endpoints aren't determined at the moment.